devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/core/tooling/dockerfiles/base/src/main/docker/Dockerfile.tpl.qute.native-distroless

```diff
@@ -0,0 +1,23 @@
+####
+# This Dockerfile is used in order to build a distroless container that runs the Quarkus application in native (no JVM) mode
+#
+# Before building the container image run:
+#
+# {buildtool.cli} {buildtool.cmd.package-native}
+#
+# Then, build the image with:
+#
+# docker build -f src/main/docker/Dockerfile.native-distroless -t quarkus/{project.artifact-id} .
+#
+# Then run the container using:
+#
+# docker run -i --rm -p 8080:8080 quarkus/{project.artifact-id}
+#
+###
+FROM {dockerfile.native-distroless.from}
+COPY {buildtool.build-dir}/*-runner /application
+
+EXPOSE 8080
+USER 1001
```

Comment:
There are only 3 users in distroless passwd file: root (0), nobody (65534) and nonroot (65532)
https://github.com/GoogleContainerTools/distroless/blob/master/base/testdata/base.yaml#L40

Reply:
nonroot should be good. 
nobody might be too limited.

Reply:
✅